### DISCUSSION TITLE
Dependency question

### DISCUSSION THREAD

**Discussion by Pavel-Sachivkin** [codingjoe/joeflow#91] [Q&A]
Hello, can you please advise if your library can be used without using Dramatiq or Celery and Redis?

**Comment by codingjoe** [comment]
Hi @Pavel-Sachivkin,

Yes, that is possible. You can implement a different task runner. However, I would not advice to run joeflow synchronously. But you could use Python's native event loop, which doesn't require any third party packages.

You can find out more about the [`JOEFLOW_TASK_RUNNER`](https://joeflow.readthedocs.io/en/latest/settings.html#joeflow.conf.JoeflowAppConfig.JOEFLOW_TASK_RUNNER) setting, in our docs.

Please let me know if you have any further questions.

Cheers!
Joe
